pkg/genericapiserver/server/genericapiserver_test.go

```diff
@@ -133,7 +133,7 @@ func TestInstallAPIGroups(t *testing.T) {
 			}, nil
 		}
 
-		mapper := api.NewDefaultRESTMapperFromScheme([]schema.GroupVersion{gv}, interfacesFor, "", sets.NewString(), sets.NewString(), scheme)
```

Comment:
Why do we still have NewDefaultRESTMapperFromScheme in pkg/api in the main repo?